src/components/cart/cart.tsx

```diff
@@ -29,32 +25,30 @@ export const Cart = ({ className }: CartProps) => {
     }
 
     return (
-        <div className={className}>
-            <Drawer title="Cart" onClose={() => setIsOpen(false)} isOpen={isOpen}>
-                {isEmpty ? (
-                    <div className={styles.emptyCart}>Cart is empty</div>
-                ) : (
-                    <div className={styles.cart}>
-                        <div className={styles.items}>
-                            {cart?.lineItems?.map((item) => (
-                                <CartItem key={item._id} cartItem={item} />
-                            ))}
-                        </div>
-                        <div className={styles.subtotalCheckout}>
-                            <label className={styles.subtotalLabel}>
-                                <span>Subtotal:</span>
-                                {cartTotals?.priceSummary?.subtotal?.formattedConvertedAmount}
-                            </label>
-                            <button
-                                className={Classnames(commonStyles.primaryButton, styles.checkout)}
-                                onClick={checkout}
-                            >
-                                Checkout
-                            </button>
-                        </div>
+        <Drawer title="Cart" onClose={() => setIsOpen(false)} isOpen={isOpen}>
```

Comment:
how could it prevent anything?

Reply:
not sure, but it just preventing :-)

Reply:
got it. so, this div wraps `Drawer` component. `Drawer` component has `position: fixed`. So, it does not stretching parent component (our div).
now, in codux, `Drawer` component is a child of that div. in order to select it, you have to click on that child inside parent, but parent has height: 0

where is a possibility to fix this (potential bug?) in codux or to set width and height to full available space for that div. but because it is not used anywehre (no className passed as a prop) - I decided to remove it